colanderalchemy/schema.py

```diff
@@ -70,42 +70,60 @@ class MyModel(Base):
                                                  'description': 'Sample'}
                    ...
 
-           The ``unknown`` argument passed to :class:`colander.Mapping`, which
```

Comment:
Yep, it was just moved down a bit to specifically under the ``unknown`` heading, line 107 and beyond.